protocol/0029-FEES-fees.md

```diff
@@ -184,9 +184,29 @@ For example, Ether is 18 decimals (wei). The smallest unit, non divisible is 1 w
 - A market is set with [Position Decimal Places" (PDP)](0052-FPOS-fractional_orders_positions.md) set to 2. A market order of size 1.23 is placed which is filled at VWAP of 100. We have fee_factor[infrastructure] = 0.001, fee_factor[maker] = 0.002, fee_factor[liquidity] = 0.05. The total fee charged to the party that placed this order is `1.23 x 100 x (0.001 + 0.002 + 0.05) = 6.519` and is correctly transferred to the appropriate accounts / pools. (<a name="0029-FEES-013" href="#0029-FEES-013">0029-FEES-013</a>). For product spot: (<a name="0029-FEES-021" href="#0029-FEES-021">0029-FEES-021</a>)
 - A market is set with [Position Decimal Places" (PDP)](0052-FPOS-fractional_orders_positions.md) set to -2. A market order of size 12300 is placed which is filled at VWAP of 0.01. We have fee_factor[infrastructure] = 0.001, fee_factor[maker] = 0.002, fee_factor[liquidity] = 0.05. The total fee charged to the party that placed this order is `12300 x 0.01 x (0.001 + 0.002 + 0.05) = 6.519` and is correctly transferred to the appropriate accounts / pools. (<a name="0029-FEES-014" href="#0029-FEES-014">0029-FEES-014</a>). For product spot: (<a name="0029-FEES-022" href="#0029-FEES-022">0029-FEES-022</a>)
 
+- During opening auction, there should be no fees collected.(<a name="0029-FEES-036" href="#0029-FEES-036">0029-FEES-036</a>)
```

Comment:
This doesn't make sense, the next AC contradicts it. Should probably be deleted.

Reply:
this is mentioned in the spec 0029 line 152 as below:
During an opening auction of a market, no fees are collected.

I guess we should should update it to:
During an opening auction of a market, no maker fees are collected.

Reply:
Ah ok, I guess the title of the section is misleading then ("Normal Auctions (including market protection and opening auctions)"). I don't know what the intendend behaviour is.